projects/igniteui-angular/src/lib/directives/toggle/README.md

```diff
@@ -49,6 +49,8 @@ handlers when the toggle is opened and respectively closed.
 | `close` | --- | `void` | Closes the toggle. |
 | `toggle` | overlaySettings?: `OverlaySettings` | `void` | Closes the toggle. |
 | `reposition` | --- | `void` | Repositions the toggle. |
+| `setOffset`  | Repositions the component in the horizontal and/or vertical directions. |deltaX, deltaY |
```

Comment:
Repositions the content - we may render ElementRef or Component. Please change component with content everywhere :)